Resources/views/app/default/_form_modal.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends '@App/default/_form_modal.html.twig' %}
+
+{% block modal_title %}
+    {# Ugly hack to change the generic title on timesheet_create modal #}
+    {% if title == 'Create' or 'Opret' %}
```

Comment:
Use https://twig.symfony.com/doc/3.x/templates.html#containment-operator here: 

```suggestion
    {% if title in ['Create', 'Opret'] %}
```

Reply:
Changed it. Thanks for the more clean solution.